### ISSUE TITLE
problems with parser in lookup nodes

### ISSUE THREAD

**Issue by evilebottnawi** [glayzzle/php-parser#184]
Input:
```php
$var
    // Hello, I am comment.
    .constant('API_URL', 'http://localhost:8080/api');
```

`what` of `propertylookup` doesn't contain `trailingComments`, (how it do babylon https://astexplorer.net/#/gist/fc79b98213d8908bf6ec3a2ab352aff4/17973de0918553d42a9312a772f8c6242ed7e1e5)

**Comment by ichiriac** [comment]
also 

```php
<?php 
// ok
$var
    // Hello, I am comment.
    . 
    // foo
constant(
    /* foo */ 'API_URL', 
    /* bar */ 'http://localhost:8080/api'
);
// ok
echo $foo;
```

**Comment by ichiriac** [comment]
fixed in #189 - released in 3.0.0-prerelease.5
